event-publisher-spi/src/main/scala/uk/gov/nationalarchives/eventpublisherspi/EventPublisherProvider.scala

```diff
@@ -26,10 +28,23 @@ class EventPublisherProvider(config: EventPublisherConfig, session: KeycloakSess
 
   override def close(): Unit = { }
 
+  def publishEvent(f: Event => String, event: Event): Unit = {
+    snsUtils.publish(f(event), config.snsTopicArn)
+  }
+
   def publishAdminEvent(f: AdminEvent => String, event: AdminEvent): Unit = {
     snsUtils.publish(f(event), config.snsTopicArn)
   }
 
+  private def accountDisabledMessage(event: Event): String = {
+    val userId = event.getUserId()
+    val realm = session.realms().getRealm(event.getRealmId())
+    val user = session.users().getUserById(realm, userId)
+    val message = s"User ${user.getUsername}" +
```

Comment:
I'm wondering whether we should be using the `user.getUsername` in the message, as this is the user's email address. Wondering about data protection, as this will be published on an open Slack channel. Might be better to use combination of `user.getFirstName` and `user.getLastName` instead.

Would be worth asking about this to the rest of the team.